src/applications/personalization/common/components/devtools/DevToolsLoader.jsx

```diff
@@ -148,23 +104,22 @@ export const DevToolsLoader = ({
 
   return (
     <div className="devtools-container">
-      <button
-        type="button"
-        className="devtools-show-button vads-u-background-color--primary vads-u-color--white"
-        onClick={handlers.togglePanel}
-      >
-        <i className="fas fa-code" />
-      </button>
+      {showIcon && (
+        <button
```

Comment:
Is there a reason for not using va-button?

Reply:
I think I had issues with the first tests with the va-button. I'll revisit that and see if I can move to use the va-button instead. Thanks